python/lsst/ts/hexrotcomm/structs.py

```diff
@@ -44,10 +47,26 @@ class Command(ctypes.Structure):
     ]
 
 
+class CommandStatus(ctypes.Structure):
+    """Command status from a Moog controller.
+
+    Called ``commandStatusStructure_t`` in the Moog controller.
+    """
+
+    _pack_ = 1
+    _fields_ = [
+        ("counter", ctypes.c_uint),
+        ("status", ctypes.c_uint),  # called cmdStatus in the Moog controller.
+        ("duration", ctypes.c_double),
+        ("reason", ctypes.c_char * COMMAND_STATUS_REASON_LEN),
+    ]
+    FRAME_ID = 0x1
```

Comment:
I am not sure this part is right or wrong. You asked me to add the `header` in the last week and the structure should be:
```
typedef struct __attribute__((__packed__)) _commandStatusStructure {
    // Header of packet
    telemetryHeaderStructure_t header;
    // Command status (enum: CmdStatus)
    unsigned int cmdStatus;
    // Estimated duration of command execution in second. The value of 0 means
    // the command excution is supposed to be fast instead of "zero" time.
    double duration;
    // Reason of the failure of command
    char reason[LENGTH_CMD_STATUS_REASON];
} commandStatusStructure_t;
```

The `counter` is from the `header` actually. If you have any case to test this, maybe you need to do the related modification.

Reply:
My mistake. I updated the code to use the counter field from the header and deleted it from the command status struct.